hypervisor/arch/x86/irq.c

```diff
@@ -151,43 +151,18 @@ static void disable_pic_irq(void)
 	pio_write8(0xffU, 0x21U);
 }
 
-static bool
-irq_desc_append_dev(struct irq_desc *desc, void *node, bool share)
+static void 
+irq_desc_append_dev(struct irq_desc *desc, void *node)
```

Comment:
append irq or dev? We don't have device concept

Reply:
it is more like to register a handler?

Reply:
replace node with action? and should not be void *

Reply:
This function would be removed completely in the second patch, so I hadn't paid much attention to the name. 
If you insist, I will change the name to set_irq_action(struct irq_desc *desc, void *node)